[TASK] Explain the extension name (#741)
TYPO3-Documentation/tea: Documentation/Introduction.rst

practices in automated code checks, unit/functional/acceptance testing and
continuous integration (CI).

.. _why-is-it-called-tea?:

> Why is it called tea?

Reviewer: ```suggestion
Why is it called "tea"?
```

Reviewer 2: ```suggestion
Why is this extension called "tea"?
```

> =====================

Reviewer: ```suggestion
=======================
```

Good extension names cover the domain of their purpose.
This extension is about managing teas. That's why the name "tea" is a good fit.

> We could add ``example_`` to the name in order to state that this extension is an example.

Reviewer 2: ```suggestion
We could also have added `_example` to the name in order to state that this extension is an example.
```

> We could also add `example_` to the name in order to state that this extension is an example.

Reviewer 2: ```suggestion
We could also add `_example` to the name in order to state that this extension is an example.
```
